nectar/downloaders/threaded.py

```diff
@@ -319,7 +319,7 @@ def _fetch(self, request):
             self.fire_download_progress(report)
 
         except SkipLocation:
-            _logger.debug("Skipping {url} because {netloc} could not be reached.".format(
+            _logger.info("Skipping {url} because {netloc} could not be reached.".format(
```

Comment:
For this case, I think it would be better to log one time that all URLs at ```netloc``` will be skipped, rather than doing it potentially thousands of times here. So that log statement would be in the code that actually makes a decision about putting ```netloc``` on a blacklist.